app/src/org/commcare/preferences/CommCareServerPreferences.java

```diff
@@ -88,6 +88,10 @@ public static String getSupportEmailAddress() {
         return getServerProperty(PREFS_SUPPORT_ADDRESS_KEY, CommCareApplication.instance().getString(R.string.support_email_address_default)) ;
     }
 
+    public static String getDataServerKey() {
```

Comment:
This can potentially be removed, depending on the answer to my question in `HttpRequestGenerator` about why this is getting used.

Reply:
It's actually getting utilized now. I crated the function earlier while forgetting to use this in actual implementation.